ArduCopter/position_vector.cpp

```diff
@@ -10,22 +10,25 @@
 // pv_location_to_vector - convert lat/lon coordinates to a position vector
 Vector3f Copter::pv_location_to_vector(const Location& loc)
 {
-    const struct Location &origin = inertial_nav.get_origin();
+    struct Location origin;
```

Comment:
if the ahrs.get_origin() fails here (which will likely never happen) then we will use the "origin" value as-is.  It hasn't been explicitly initialised but I suspect we always initialise new variables to zero so it's probably ok.